front/src/modules/ui/chip/components/EntityChip.tsx

```diff
@@ -83,7 +83,7 @@ export function EntityChip({
     navigate(linkToEntity);
   }
 
-  return clickable && linkToEntity ? (
+  return clickable && linkToEntity && name.trim() ? (
```

Comment:
could you you use our `isNonEmptyString` guard instead?
(if the name is " ", I feel we should not consider it as empty (it means somebody as purposefully set it as " " ==> trim() does not seem to be what we want here)

Reply:
i can use that guard. so you saying for " " case, it should show avatar?

> could you you use our `isNonEmptyString` guard instead? (if the name is " ", I feel we should not consider it as empty (it means somebody as purposefully set it as " " ==> trim() does not seem to be what we want here)

Reply:
i sent a commit for guard usage and removed trim.